contracts/KetlAttestation.sol

```diff
@@ -133,10 +134,24 @@ contract KetlAttestation is ERC1155, Ownable, Versioned, ERC2771Recipient {
     minimumEntanglementCounts[_id] = _minimumEntanglementCount;
   }
 
+  function setMaximumEntanglementsPerAttestation(
+    uint _maximumEntanglementsPerAttestation
+  ) public onlyOwner {
+    maximumEntanglementsPerAttestation = _maximumEntanglementsPerAttestation;
+  }
+
   function setCurrentTokenId(uint32 _currentTokenId) public onlyOwner {
     currentTokenId = _currentTokenId;
   }
 
+  function attestationHashesEntangled(
```

Comment:
We shouldn't try to maintain dynamic code bases (the ones that are currently actively being worked on) compatibility by satisfying the legacy code with maintaining the interfaces. I recommend ditching the `attestationHashesEntangled` interface and instead creating PRs at the repos that depend on this piece of code.

It will require a bit of orchestration in the short term but will also help a lot in the long term.

Reply:
Updated to remove this.